tests/unit/service_test.py

```diff
@@ -331,6 +331,13 @@ def test_create_container_no_build(self):
         service.create_container(do_build=False)
         self.assertFalse(self.mock_client.build.called)
 
+    def test_create_container_no_build_cgroup_parent(self):
+        service = Service('foo', client=self.mock_client, build='.')
+        service.image = lambda: {'Id': 'abc123'}
+
+        service.create_container(do_build=False, cgroup_parent='test')
+        self.assertFalse(self.mock_client.build.called)
```

Comment:
I don't see how this test is testing that our `cgroup_parent` option was passed in correctly. I think this is the the wrong place to write the test for the option.

Maybe something along the lines to this test, https://github.com/docker/compose/blob/master/tests/unit/service_test.py#L111
where we can see that the `cgroup_parent`, if specified, is sent to create_host_config.